src/components/Connection/Connection.module.css

```
@@ -4,6 +4,8 @@
   color: #ffffff;
   display: flex;
   flex-direction: column;
+  padding-top:40px;
+  padding-bottom:40px;
```

add task to the backlog: add linter to decentraland-auth

```suggestion
  padding-top: 40px;
  padding-bottom: 40px;
```

---

Fixed manually.